### PR TITLE
Details and upsert

### PR DESCRIPTION
**Pull request by Pierrelgx**
Update component Upsert

### CODE REVIEW

File: src/components/TodoUpsert/TodoUpsert.styles.tsx [hunk 1]
```diff
@@ -5,16 +5,17 @@ export const TodoUpsertLabel = styled.label`
...
 `;
 
 export const Input = styled.input`
     background-color: ${(props) => props.theme.colors.spanishGray20};
     color: ${(props) => props.theme.colors.cultured};
-    border-radius: 4px;
-    padding: 6px;
+    border-radius: 8px;
+    padding: 12px;
+		font-weight: ${(props) => props.theme.fontWeights.regular} ;
```
Comment (Lambtsa) [inline src/components/TodoUpsert/TodoUpsert.styles.tsx:18]: Fix indentation

File: src/components/TodoDetails/TodoDetails.tsx [hunk 2]
```diff
@@ -1,14 +1,38 @@
+// import { Todo } from "components/Todo";
+// import { TodoDescription } from "components/Todo/Todo.styles";
```
Comment (Lambtsa) [inline src/components/TodoDetails/TodoDetails.tsx]: Remove and clean up all over the app. Any comments that are not needed anymore, imports that are not needed or styles that are not used.

File: src/components/TodoList/Todolist.styles.tsx [hunk 3]
```diff
@@ -1,9 +1,4 @@
 import styled from "styled-components";
 
 export const TodoListWrapper = styled.div`
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
-  gap: 16px;
 `;
```
Comment (Lambtsa) [inline src/components/TodoList/Todolist.styles.tsx]: Remove whole file if not needed

File: src/components/TodoUpsert/TodoUpsert.styles.tsx [hunk 4]
```diff
@@ -1,3 +1,25 @@
...
+    gap: 8px;
+
+`;
+
+export const Input = styled.input`
+    background-color: ${(props) => props.theme.colors.spanishGray20};
+    color: ${(props) => props.theme.colors.cultured};
+    border-radius: 8px;
+    padding: 12px;
+		font-weight: ${(props) => props.theme.fontWeights.regular} ;
```
Comment (Lambtsa) [inline src/components/TodoUpsert/TodoUpsert.styles.tsx:18]: Indentation

File: src/components/TodoUpsert/TodoUpsert.tsx [hunk 5]
```diff
@@ -7,6 +9,27 @@
 export const TodoUpsert = (): JSX.Element => {
   return (
     <>
```
Comment (Lambtsa) [inline src/components/TodoUpsert/TodoUpsert.tsx]: Remove unnecessary fragments (<></>)

**Review by Lambtsa — CHANGES_REQUESTED** [review]
Go through the different things you have added and remove comments and clean up code

File: src/components/TodoUpsert/TodoUpsert.styles.tsx (hunk 1 above)
Comment (Pierrelgx) [reply src/components/TodoUpsert/TodoUpsert.styles.tsx:18]: done

File: src/components/TodoDetails/TodoDetails.tsx (hunk 2 above)
Comment (Pierrelgx) [reply src/components/TodoDetails/TodoDetails.tsx]: done

File: src/components/TodoList/Todolist.styles.tsx (hunk 3 above)
Comment (Pierrelgx) [reply src/components/TodoList/Todolist.styles.tsx]: done

File: src/components/TodoUpsert/TodoUpsert.styles.tsx (hunk 4 above)
Comment (Pierrelgx) [reply src/components/TodoUpsert/TodoUpsert.styles.tsx:18]: done

File: src/components/TodoUpsert/TodoUpsert.tsx (hunk 5 above)
Comment (Pierrelgx) [reply src/components/TodoUpsert/TodoUpsert.tsx]: done

**Review by Lambtsa — APPROVED** [review]

**Pull request merged**